--- lib/python3.10/site-packages/numpy/lib/utils.py
+++ lib/python3.10/site-packages/numpy/lib/utils.py
@@ -43,6 +43,11 @@
         # using installed numpy core headers
         import numpy.core as core
         d = os.path.join(os.path.dirname(core.__file__), 'include')
+    d = os.path.normpath(d)
+    import sysconfig
+    crosspath = sysconfig.get_path('platlib')
+    buildpath = crosspath.replace('/cross/lib', '/build/lib')
+    d = d.replace(buildpath, crosspath)
     return d
 
 
