diff --git a/npymath.ini b/npymath.ini
index 3e465ad..077cbd0 100644
--- a/npymath.ini
+++ b/npymath.ini
@@ -4,10 +4,9 @@ Description=Portable, core math library implementing C99 standard
 Version=0.1
 
 [variables]
-pkgname=numpy.core
-prefix=${pkgdir}
-libdir=${prefix}/lib
-includedir=${prefix}/include
+# Force it to find cross-build libs when we build scipy
+libdir=@LIBDIR
+includedir=@INCDIR
 
 [default]
 Libs=-L${libdir} -lnpymath
